tests/e2e/tasks/tenant/run_workload.yml

```diff
@@ -24,6 +24,13 @@
         replace: "boot_disk_copy: true"
       when: set_boot_disk_copy|default(false)|bool
 
+    - name: set data copy
+      ansible.builtin.replace:
+        path: "{{ os_migrate_data_dir }}/workloads.yml"
+        regexp: "data_copy: false"
+        replace: "data_copy: true"
+      when: set_data_copy|default(true)|bool
```

Comment:
This should be in the inverse -- we should be looking for "data_copy: true" and replacing it with "data_copy: false". So the variable name should probably be `set_no_data_copy` or `set_data_copy_false` or `unset_data_copy`.